### PR TITLE
add issue filter by assignee and label in projects

### PR DESCRIPTION
**Pull request by iminfinity**
Partial implementation of https://github.com/go-gitea/gitea/issues/21846

this extends on https://github.com/go-gitea/gitea/pull/21963 by @techknowlogick

---
close #21963 

---
*PR sponsored by [Obmondo.com](https://obmondo.com)*

### CODE REVIEW

**Comment by 6543** [comment]
Why not also add this filter option to repo projects?

**Comment by iminfinity** [comment]
> Why not also add this filter option to repo projects?

@6543  we only needed this filter in  org/-/projects in our org, but i can try  to add it to repo/projects if it is required

**Comment by 6543** [comment]
ping me if you finished it :)

**Comment by iminfinity** [comment]
> ping me if you finished it :)

@6543

File: templates/projects/view.tmpl [hunk 1]
```diff
@@ -59,6 +59,60 @@
 
 <div class="content">{{$.Project.RenderedContent|Str2html}}</div>
 
+<div class="issue-list-toolbar-right" id="org-projects">
+	<div class="ui secondary filter menu labels">
+		<!-- Label -->
+		<div class="ui {{if (and (not .Labels) (not .OrgLabels)) }}disabled{{end}} dropdown jump item label-filter">
```
Comment (6543) [inline templates/projects/view.tmpl]:
```suggestion
		<div class="ui {{if (and (not .Labels) (not .OrgLabels))}}disabled{{end}} dropdown jump item label-filter">
```

(fix linter issue)

**Comment by 6543** [comment]
please dont forcepush just merge target branch in and push new commits ontop to it

we do squash-merges :)

**Comment by 6543** [comment]
@iminfinity could you please the linter?

**Comment by iminfinity** [comment]
> @iminfinity could you please the linter?

`make lint` and `make lint-frontend` did not catch that

**Comment by 6543** [comment]
could you update the branch :point_up_2: and the screenrecord is not accesable anymore :/

**Comment by iminfinity** [comment]
> could you update the branch 👇 and the screenrecord is not accesable anymore :/

https://github.com/go-gitea/gitea/assets/20443971/4d307e38-a9af-4ad9-b99e-0ee059873b8f



i can still play the old screen record though

**Comment by iminfinity** [comment]
ahh looks like i made the label filter `or` instead of `and` in the new update

**Pull request closed without merging**